labs/06/3d_recognition.py

```diff
@@ -36,9 +37,14 @@ def next_batch(self, batch_size):
         batch_perm, self._permutation = self._permutation[:batch_size], self._permutation[batch_size:]
         return self._voxels[batch_perm], self._labels[batch_perm]
 
+    def next_batch_data_for_prediction(self, batch_size):
```

Comment:
Please get rid of the next_batch_data_for_prediction; instead, make `next_batch` return None for labels, i.e., something like
  return self._voxels[batch_perm], self._labels[batch_perm] if self._labels is not None else None